EfEnumToLookup/LookupGenerator/SqlServerHandler.cs

```diff
@@ -111,12 +117,12 @@ private string PopulateLookup(LookupData lookup)
 	WHEN MATCHED AND src.Name <> dst.Name THEN
 		UPDATE SET Name = src.Name
 	WHEN NOT MATCHED THEN
-		INSERT (Id, Name)
-		VALUES (src.Id, src.Name)
+		INSERT (Id, Name, Description)
```

Comment:
this indicates we could do with extending the test suite to cover add/remove/updating the lookup rows in the db as the model is changed.

plan: run apply, directly manipulate the db to add/edit/remove, assert changed (just to avoid false positive if this step silently failed), run apply again, assert that table was returned to original state